lib/autocomplete-manager.js

```diff
@@ -138,16 +142,22 @@ export default class AutocompleteManager {
     }
   }
 
+  watchEditor (editor, labels) {
```

Comment:
Yes, this will be the public method. I tried it out locally in `find-and-replace` but I forgot to add the test. Added those now.